kolibri_exercise_perseus_plugin/generateKATranslations.js

```diff
@@ -0,0 +1,288 @@
+/*
+ * This utility is designed to download Khan Academy po files from crowdin,
+ * search through them for interface strings that we need in the Perseus renderer,
+ * and then upload them to our Crowdin in order to leverage these translated strings
+ * in our own translations.
+ */
+
+
+/*
+ * Utility functions for generating the paths for locale files, both locally and on crowdin
+ */
+const localeFile = (lang, locale) => {
+  if (locale) {
+    if (locale.length <= 2) {
+      locale = locale.toUpperCase();
+    } else {
+      locale = locale[0].toUpperCase() + locale.slice(1);
+    }
+  }
+  const langCode = lang + (locale ? '_' + locale : '');
+  return `${langCode}/LC_FRONTEND_MESSAGES/exercise_perseus_render_module-messages.json`;
+};
+
+// Format the language code how crowdin expects it
+const crowdinLangCode = (lang, locale) => {
+  return lang + (locale ? '-' + locale.toUpperCase() : '');
+};
+
+// Path to the locale file for a language in the local locale folder
+const localLocaleFile = (lang, locale) => {
+  return './locale/' + localeFile(lang, locale);
+}
+
+// Which kolibri branch to upload translations to on Crowdin
+const kolibriReleaseBranch = 'release-v0.6.x';
```

Comment:
Yes, basically it maps to where this version of the exercise renderer is going to submit and receive its translations from.